### ISSUE TITLE
Request for Commands

### ISSUE THREAD

**Issue by crazybmanp** [crazybmanp/RyuZU-Bot#5]
lewd, ecchi-sketch, nsfw, loli
=neko, =waa
Marriage
divorce, marry, acceptmarriage, declinemarriage,
smug
slap

Official list:
- [ ] lewd
- [ ] waa
- [ ] smug
- [x] neko

**Comment by crazybmanp** [comment]
- [ ] lewd
- [ ] neko
- [ ] waa
- [ ] smug
- [ ] slap

**Comment by crazybmanp** [comment]
@razage should we uninclude the slap functionality. i don't see much point.

**Comment by razage** [comment]
Yeah I don't really see a reason to add slap. Further down the line, we could just add an emote cog or something and put stupid shit like that there.

**Comment by razage** [comment]
Notably everything left that was requested requires either:
* An API that delivers them
* A folder of pictures we supply

I personally do not own a collection of lewd or waa reactions. I do have a smug folder though.

**Comment by crazybmanp** [comment]
i don't know if i feel ok hosting a collection of lewd reactions
